integration-cli/docker_cli_events_test.go

```diff
@@ -77,25 +70,18 @@ func (s *DockerSuite) TestEventsContainerFailStartDie(c *check.C) {
 
 	out, _ := dockerCmd(c, "images", "-q")
 	image := strings.Split(out, "\n")[0]
-	if _, _, err := dockerCmdWithError("run", "--name", "testeventdie", image, "blerg"); err == nil {
-		c.Fatalf("Container run with command blerg should have failed, but it did not")
-	}
+	_, _, err := dockerCmdWithError("run", "--name", "testeventdie", image, "blerg")
+	c.Assert(err, checker.NotNil) //Container run with command blerg should have failed, but it did not
 
 	out, _ = dockerCmd(c, "events", "--since=0", fmt.Sprintf("--until=%d", daemonTime(c).Unix()))
 	events := strings.Split(out, "\n")
-	if len(events) <= 1 {
-		c.Fatalf("Missing expected event")
-	}
+	c.Assert(len(events), checker.GreaterThan, 1) //Missing expected event
```

Comment:
@cpuguy83 didn't see the comment. It does not work as of now, but that's an idea :wink: